src/Row.tsx

```diff
@@ -81,6 +81,7 @@ function Row<R, SR = unknown>({
             isRowSelected={isRowSelected}
             eventBus={eventBus}
             dragHandleProps={isCellSelected ? (selectedCellProps as SelectedCellProps)!.dragHandleProps : undefined}
+            onFocus={isCellSelected ? (selectedCellProps as SelectedCellProps)!.onFocus : undefined}
```

Comment:
non-nullish assertions are redundant.
```suggestion
            dragHandleProps={isCellSelected ? (selectedCellProps as SelectedCellProps).dragHandleProps : undefined}
            onFocus={isCellSelected ? (selectedCellProps as SelectedCellProps).onFocus : undefined}
```